src/index.ts

```diff
@@ -1,19 +1,35 @@
 import express, { Express, Response, Request } from "express";
+import mongoose, { ConnectOptions } from "mongoose";
 import dotenv from "dotenv";
 
+// Initialize configuration from .env file
 dotenv.config();
 
+// Express app setup
 const app: Express = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 3000; // Server port
 
+// Database connection
+const databaseUrl = process.env.DATABASE_URL || ""; // Ensure DATABASE_URL is defined
+mongoose
+  .connect(databaseUrl, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  } as ConnectOptions)
+  .then(() => console.log("Connected to MongoDB Atlas"))
+  .catch((err) => console.error("Could not connect to MongoDB Atlas", err));
+
+// Basic route for home page
 app.get("/", (_, res: Response) => {
   res.send("Hello World!");
 });
 
+// Echo function to display request URLs
 function echo(req: Request, res: Response) {
   res.send(req.originalUrl);
 }
```

Comment:
Sorry I didn't make this clear, but this function was just a placeholder for future implementation. You can delete this now